### PR TITLE
Notebooks protobuildings

### PR DESCRIPTION
**Pull request by ThibaultMarzullo**
#### Any background context you want to provide?

The Documentation contains Jupyter Notebooks that walk users through the creation of a BSync XML using bsyncpy.
The existing example of a L100 audit in a DOE Prototype small office building was expanded to include a L200 audit of the same building, and both a L100 and L200 audit of the DOE Prototype primary school building.

#### What does this PR do?

It adds the L100 and L200 examples for the DOE Prototype primary school building

#### How should this be manually tested?

Review the notebook. Generate the two .xml files. Replace the XML headers to specify the correct namespaces. Upload them to the Schema Validator.

#### What are the relevant tickets?

[Reference implementations of 2 DOE prototype buildings in BuildingSync 2.4](https://github.com/BuildingSync/project-tracker/issues/85)

#### Screenshots (if appropriate)

Screen Shot 2022-06-24 at 4 09 04 PM

### CODE REVIEW

**Comment by ThibaultMarzullo** [comment]
@nllong @JieXiong9119 , I have some cleaning up to do on that. Can you verify the code part and kindly ignore the text for the moment? I have some descriptions to write near the end of the document and some section numbers to verify.
Also, POM values are made up and I will update with simulated values.
Thanks!

**Comment by ThibaultMarzullo** [comment]
@laurenliz22 , the text has been updated! I think it is ready for review

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 1]
```diff
@@ -0,0 +1,8205 @@
...
+
+1) Setting up the root, or the base, of your BuildingSync file
+
+2) Adding the information required for a Level 1 energy audit
+
+3) Adding the information required for a Level 2 energy audit
+
+4) Adding the information required for a Level 3 energy audit (WIP)
+
+5) Generating a BuildingSync file and verifying it against version 2.4.0 of the BuildingSync schema
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:32]:
```suggestion
4) Generating a BuildingSync file and verifying it against version 2.4.0 of the BuildingSync schema
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 2]
```diff
@@ -0,0 +1,8205 @@
...
+
+
+This looks exactly like it was supposed to look. If you needed to add a different site, you would just need to add a child to the appropriate parent, in that case a Site to a group of Sites.
+Now, let us carry on and start the Level 1 audit.
+
+## 5. Level 1 audit
+
+We refer to ASHRAE Standard 211, section 5.3, relevant to the Level 1 energy audit procedures.
+For this audit, we will need to report:
+1) a facility description (section 6.1.1.1)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:159]:
```suggestion
1) a facility description (section 6.1.1.1) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 3]
```diff
@@ -0,0 +1,8205 @@
...
+2) the historical energy use (section 6.1.1.2)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:160]:
```suggestion
2) the historical energy use (section 6.1.1.2) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 4]
```diff
@@ -0,0 +1,8205 @@
...
+3) a benchmark of the site EUI (section 6.1.1.3)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:161]:
```suggestion
3) a benchmark of the site EUI (section 6.1.1.3) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 5]
```diff
@@ -0,0 +1,8205 @@
...
+4) the target and estimate savings (section 6.1.1.4)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:162]:
```suggestion
4) the target and estimate savings (section 6.1.1.4) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 6]
```diff
@@ -0,0 +1,8205 @@
...
+5) low-cost and no-cost energy efficiency measure recommendations (section 6.1.1.5)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:163]:
```suggestion
5) low-cost and no-cost energy efficiency measure recommendations (section 6.1.1.5) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 7]
```diff
@@ -0,0 +1,8205 @@
...
+6) potential capital energy efficiency measure recommendations (section 6.1.1.6)
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:164]:
```suggestion
6) potential capital energy efficiency measure recommendations (section 6.1.1.6) <br>
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 8]
```diff
@@ -0,0 +1,8205 @@
...
+)
+mech_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+mech_tous += mech_tou_hpw
+mech_tous += mech_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:564]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 9]
```diff
@@ -0,0 +1,8205 @@
...
+)
+lib_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+lib_tous += lib_tou_hpw
+lib_tous += lib_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:617]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 10]
```diff
@@ -0,0 +1,8205 @@
...
+)
+kitchen_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+kitchen_tous += kitchen_tou_hpw
+kitchen_tous += kitchen_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:670]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 11]
```diff
@@ -0,0 +1,8205 @@
...
+)
+gym_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+gym_tous += gym_tou_hpw
+gym_tous += gym_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:723]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 12]
```diff
@@ -0,0 +1,8205 @@
...
+)
+classroom1_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+classroom1_tous += classroom1_tou_hpw
+classroom1_tous += classroom1_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:937]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 13]
```diff
@@ -0,0 +1,8205 @@
...
+)
+classroom2_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+classroom2_tous += classroom2_tou_hpw
+classroom2_tous += classroom2_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:988]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 14]
```diff
@@ -0,0 +1,8205 @@
...
+)
+classroom3_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+classroom3_tous += classroom3_tou_hpw
+classroom3_tous += classroom3_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1039]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 15]
```diff
@@ -0,0 +1,8205 @@
...
+)
+corridor1_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+corridor1_tous += corridor1_tou_hpw
+corridor1_tous += corridor1_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1094]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 16]
```diff
@@ -0,0 +1,8205 @@
...
+)
+corridor2_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+corridor2_tous += corridor2_tou_hpw
+corridor2_tous += corridor2_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1145]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 17]
```diff
@@ -0,0 +1,8205 @@
...
+)
+corridormain_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+corridormain_tous += corridormain_tou_hpw
+corridormain_tous += corridormain_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1247]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 18]
```diff
@@ -0,0 +1,8205 @@
...
+)
+lobby_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+lobby_tous += lobby_tou_hpw
+lobby_tous += lobby_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom, lobby or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1300]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 19]
```diff
@@ -0,0 +1,8205 @@
...
+)
+wb_tou_wpy = bsync.TypicalOccupantUsage(
+    bsync.TypicalOccupantUsageUnits("Weeks per year"),
+    bsync.TypicalOccupantUsageValue(50.)
+)
+wb_tous += wb_tou_hpw
+wb_tous += wb_tou_wpy
+
+# 5.3.4.d looks for normal / design or peak occupancy.
+# Here, we take values from the IDF as the building has been modelled: with no occupants in the mech room, bathroom, lobby or corridors.
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1354]:
```suggestion
# Here, we take values from the IDF as the building has been modeled: with no occupants in the mech room, lobby, bathroom or corridors.
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 20]
```diff
@@ -0,0 +1,8205 @@
...
+    bsync.OccupantQuantityType("Peak total occupants"),
+    bsync.OccupantQuantity(1849.)
+)
+wb_occ_levels += wb_occ_design
+
+# finally, we add these to the actual section element
+wb_sec += wb_fas
+wb_sec += wb_tous
+wb_sec += wb_occ_levels
+```
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1366]:
```suggestion
pretty_print(sections)
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 21]
```diff
@@ -0,0 +1,8205 @@
...
+
+```python
+# The Primary School prototype model has 3 packaged rooftop units (single-speed dx cooling and fuel fired furnace) and 4 packaged rooftop VAV with hot water reheat.
+# We create them and link them back to the sections of interest.
+# At this point, we aren't modeling separate HVAC zones.
+# This level of detail fulfills requirements for a Level 1 audit
+
+hvac_systems = bsync.HVACSystems()
+systems+=hvac_systems
+plugloads = [{'spaceid' : lobby_sec["ID"], 'plugload' : 4.0},
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1462]: do we need to define plugloads again?

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 22]
```diff
@@ -0,0 +1,8205 @@
...
+
+hvac_systems += vav_pod1
+hvac_systems += vav_pod2
+hvac_systems += vav_pod3
+hvac_systems += vav_other
+hvac_systems += pszac_1
+hvac_systems += pszac_2
+hvac_systems += pszac_3
+
+
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1579]:
```suggestion
    pretty_print(systems)
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 23]
```diff
@@ -0,0 +1,8205 @@
...
+Now that we have a quick sense of the building, let's start off by looking at the requirements for an ASHRAE 211 Level 1 audit. This leads us to the concept of a [Scenario](https://buildingsync.net/schema/v2.3.0/documentation/BuildingSync_xsd.html#ScenarioType). A Scenario in BuildingSync is used to refer to energy and timeseries data associated with a particular, well, scenario. Specifically, there are 5 primary Scenarios used in BuildingSync, all which relate back to Standard 211:
+
+![Scenario Types](./img/b1-sc-docs.png)
+
+We will repeatedly come back to the concept of a Scenario, as they are core to organizing information in a BuildingSync document. 
+
+The current building measured scenario is intended to capture true measured historical data. Typically this refers to utility bill data, but AMI type data can also be captured (later).  We start off by creating a new scenario element and defining its type as follows:
+- `Scenario[ScenarioType/CurrentBuilding/CalculationMethod/Measured]`. This is an XPath expression that can be interpreted as "A Scenario that has the child elements ScenarioType/CurrentBuilding/CalculationMethod/Measured".  The XML for this would look like:
+
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1594]:
```suggestion
```xml
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 24]
```diff
@@ -0,0 +1,8205 @@
...
+<Scenario>
+    <ScenarioType>
+        <CurrentBuilding>
+            <CalculationMethod>
+                <Measured/>
+            </CalculationMethod>
+        </CurrentBuilding>
+    </ScenarioType>
+</Scenario>
+
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1604]:
```suggestion
```
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 25]
```diff
@@ -0,0 +1,8205 @@
...
+## 6. Level 2 Audit
+
+Now that we have set up the level 1 audit, we can move on to level 2. Great!
+
+Level 2 audits include all elements from level 1 audits, and add quite a few details to them. Let us walk through the report requirements in order as they appear in ASHRAE Standard 211, Section 6.2.
+
+### 6.1 Facility Description
+
+For section 6.2.1.1, Building Information, we can use the information we collected during the level 1 audit and enrich it somewhat. 
+We will add schedules first (occupancy, lighting, process and plug loads, and equipment).
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:2335]:
```suggestion
We will add schedules first (occupancy, lighting, equipment and HVAC). 
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 26]
```diff
@@ -0,0 +1,8205 @@
...
+        )
+    ),
+    bsync.FloorCovering("Carpet"),
+    bsync.FloorConstructionType("Built up"),
+    ID="Foundation"
+)
+foundation_sys += foundation
+systems += foundation_sys
+```
+
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:5677]: Do we need to add in overall enclosure tightness as well?

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 27]
```diff
@@ -0,0 +1,8205 @@
...
+###### 6.2.3.4 Ceilings
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:5678]: Didn't see this in 211?

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 28]
```diff
@@ -0,0 +1,8205 @@
...
+        bsync.HeatingPlant.ControlSystemTypes(
+            bsync.ControlSystemType(
+                bsync.ControlSystemType.Other(
+                    bsync.OtherCommunicationProtocolName("Unknown")
+                )
+            )
+        ),
+        bsync.YearInstalled(2018),
+        bsync.PrimaryFuel('Natural gas'),
+        bsync.BuildingAutomationSystem(Tr
```
Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:6953]: should we see more of the VAV output here?

Comment (laurenliz22) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:8067]: Getting error message: "TypeError: integer expected"

**Review by laurenliz22 — COMMENTED** [review]
@ThibaultMarzullo This is awesome, thank you for all of your hard work.  I added in a few comments - do you mind looking through and letting me know if you have any questions.  Thank you!!

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md (hunk 26 above)
Comment (ThibaultMarzullo) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:5677]: I have that in the AirInfiltrationSystems near the end. Do you mean that?

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md (hunk 28 above)
Comment (ThibaultMarzullo) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:8067]: On my end everything works fine, can you tell me which particular line is expecting an integer please?

Screen Shot 2022-06-29 at 2 20 23 PM

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md (hunk 26 above)
Comment (laurenliz22) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:5677]: Ah ok - I did not realize it was the same thing,  Thanks for clarifying.

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md (hunk 28 above)
Comment (laurenliz22) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:6953]: Got it! thanks!

Comment (laurenliz22) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:8067]: Line 8067: "pom_led += bsync.AnnualDemandSavingsCost(872.57)" is where I got the error.  If i change it to 872 it worked.

Comment (ThibaultMarzullo) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:8067]: I see it now! Sorry, somehow it was showing this comment for lines 6666-6669.

**Comment by ThibaultMarzullo** [comment]
@laurenliz22 Thank you, I have committed the corrected version following your review. I have not added a ReadMe file as Jie already created a single ReadMe for the whole bsync_examples folder that mentions the primary school example.

**Review by laurenliz22 — APPROVED** [review]
Looks great!  Thanks Thibault!

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 29]
```diff
@@ -0,0 +1,8821 @@
...
+    as_etree.set("version", "2.4.0")  
+    output = etree.tostring(as_etree, doctype=doctype, pretty_print=True)
+    with open(file, 'wb+') as f:
+        f.write(output)
+        return True
+```
+
+## 4. Setting up the root of the BuildingSync file
+
+We are going to define the basic building blocks (ha!) of our representation of the primary school prototype building. For that, we need to define a buiding, which belongs to a site, which belongs to a facility, which belongs to the highest-level of our BuildingSync file: the BuildingSync root. It will look like this:
```
Comment (nllong) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:64]: haha!

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 30]
```diff
@@ -0,0 +1,8821 @@
...
+```python
+# Defining the root is as easy as that
+xmlargs = {
+    'xmlns:xsi' : "http://www.w3.org/2001/XMLSchema-instance",
+    'xsi:SchemaLocation' : "http://buildingsync.net/schemas/bedes-auc/2019 ../../../BuildingSync.xsd"
+}
+
+# xmlns:xsi = "http://www.w3.org/2001/XMLSchema-instance"
+    # xsi:SchemaLocation = "http://buildingsync.net/schemas/bedes-auc/2019 ../../../BuildingSync.xsd"
+root = bsync.BuildingSync()#**kwargs=xmlargs)
```
Comment (nllong) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:98]:
```suggestion
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 31]
```diff
@@ -0,0 +1,8821 @@
...
+
+
+```python
+# Defining the root is as easy as that
+xmlargs = {
+    'xmlns:xsi' : "http://www.w3.org/2001/XMLSchema-instance",
+    'xsi:SchemaLocation' : "http://buildingsync.net/schemas/bedes-auc/2019 ../../../BuildingSync.xsd"
+}
+
+# xmlns:xsi = "http://www.w3.org/2001/XMLSchema-instance"
```
Comment (nllong) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:96]:
```suggestion
```

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 32]
```diff
@@ -0,0 +1,8821 @@
...
+kitchen_sec += kitchen_occ_levels
+```
+
+#### 5.2.5 Gym
+
+
+```python
+# create a new section
+gym_sec = bsync.Sections.Section(ID="Gym")
+gym_sec += bsync.SectionType('Space function')
```
Comment (nllong) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:719]:
```suggestion
gym_sec += bsync.SectionType("Space function")
```
this repeats...

File: docs/notebooks/bsync_examples/Primary-School-Levels1-2.md [hunk 33]
```diff
@@ -0,0 +1,8821 @@
...
+
+```python
+systems = bsync.Systems()
+# Systems, measures, reports and contacts belong to our facility (f1)
+f1 += systems
+
+# Let us automate the task this time
+# We first create a list of dictionaries that contains information on space IDs and the appropriate plug load. The plug load is taken from the Primary School Prototype ASHRAE Standard 90.1-2016 model
+# We take the IDs from the sections we created earlier
+plugloads = [{'spaceid' : lobby_sec["ID"], 'plugload' : 4.0},
```
Comment (nllong) [inline docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1946]: good job on the space functions! I'm wondering if in the future we can create a dict or something that we store all the space function data in first, then just iterate? Similar to your space plugloads here.

Comment (ThibaultMarzullo) [reply docs/notebooks/bsync_examples/Primary-School-Levels1-2.md:1946]: Yes, agreed. I'll also do the same for schedules, it's a bit too dense right now IMHO.

**Pull request merged**